### DISCUSSION TITLE
How to set hover background color for QTab or QRouteTab?

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#11940] [General - Components / Directives / etc] [answered]
How can I set the background color when hovering over a QTab/QRouteTab? It also looks like the default hover background color is kind of dimmed. How can I get this undimmed?

This is what I currently get when I hover over my QRouteTab:

![image](https://user-images.githubusercontent.com/19785727/148535127-a72bf6f2-903d-4be5-9b30-f04daaa851af.png)

This is the background color when I have clicked the tab:

![image](https://user-images.githubusercontent.com/19785727/148535195-f322c7ed-02d1-4589-b77b-d7c1520ed51e.png)

I want the hover background color to be the same as the selected background color.

**Comment by pdanpdan** [comment, marked as answer]
Make a codepen so we can play with it
Use the inspector and check the DOM - `.q-focus-helper`, `.q-focus-helper:before`, and `.q-focus-helper:after` - adjust them as you like
See the source definition here: https://github.com/quasarframework/quasar/blob/a5994932817212ed799825144629fb7ab077da4c/ui/src/css/core/visibility.sass#L118

**Reply by eugenevk** [reply]
I was able to adjust it by creating a class plus hover in app.scss:
```
.nav-item {
    opacity: 1;
}
.nav-item:hover {
    background-color: $secondary;
}
```
and in my component:
```
<q-route-tab
  class="nav-item"
  exact
  name="timecards"
  no-caps
  :ripple="false"
  style="justify-content:initial"
  :to="{ name: 'Timecards' }"
>
  <q-icon name="fas fa-business-time" class="q-ml-sm q-mr-md" />Timecards
</q-route-tab>
```
